docs/bricks_design.rst

```diff
@@ -0,0 +1,602 @@
+
+Bricks design explained
+=======================
+
+This is a runnable tutorial, which means you can copy its fragments 
+to an IPython notebook and run. The import section follows:
```

Comment:
Just an idea - maybe make it an ipython notebook? They are easy to view right now, and no copy-paste is necessary.